src/frontend/Felles/Visittkort/Visittkort.tsx

```diff
@@ -60,6 +75,17 @@ const VisittkortComponent: FC<{
         fullmakt,
         vergemål,
     } = personopplysninger;
+    const skalLenkeTilFagsystemBehandling =
+        behandling.påklagetVedtak.påklagetVedtakstype === PåklagetVedtakstype.VEDTAK &&
+        behandling.påklagetVedtak.fagsystemVedtak?.fagsystemType === FagsystemType.ORDNIÆR;
```

Comment:
Enum'n er ikke opprettet her, men bør kanskje endre typoen på sikt 
```suggestion
        behandling.påklagetVedtak.fagsystemVedtak?.fagsystemType === FagsystemType.ORDINÆR;
```

Reply:
Helt enig der, men det må tas i en egen pr fordi det krever endringer i backend og database også